token-lending/program/src/processor.rs

```diff
@@ -818,7 +852,37 @@ fn process_deposit_obligation_collateral(
     let user_transfer_authority_info = next_account_info(account_info_iter)?;
     let clock = &Clock::from_account_info(next_account_info(account_info_iter)?)?;
     let token_program_id = next_account_info(account_info_iter)?;
+    _deposit_obligation_collateral(
+        program_id,
+        collateral_amount,
+        source_collateral_info,
+        destination_collateral_info,
+        deposit_reserve_info,
+        obligation_info,
+        lending_market_info,
+        lending_market_authority_info,
+        obligation_owner_info,
+        user_transfer_authority_info,
+        clock,
+        token_program_id,
+    )
+}
 
+#[allow(clippy::too_many_arguments)]
+fn _deposit_obligation_collateral<'a>(
```

Comment:
i guess i just wondered the purpose of splitting this one out if it was still returning program result maybe? but yeh w/e

Reply:
You'll see